src/pages/popup.html

```diff
@@ -10,6 +10,72 @@
             body {
                 width: 400px;
             }
+			/* The switch - the box around the slider */
+			.switch {
+			  position: relative;
+			  display: inline-block;
+			  width: 60px;
+			  height: 34px;
+			}
+
+			/* Hide default HTML checkbox */
+			.switch input {
+			  opacity: 0;
+			  width: 0;
+			  height: 0;
```

Comment:
```suggestion
			display: none;
```
Does this work here?